No ipdb in dev venv environment

If you're having trouble using Certbot and aren't sure you've found a bug or
request for a new feature, please first try asking for help at
https://community.letsencrypt.org/. There is a much larger community there of
people familiar with the project who will be able to more quickly answer your
questions.

## My operating system is (include version):
MacOS 12 Monterey (M1 chip)

## I installed Certbot with (snap, OS package manager, pip, certbot-auto, etc):
git clone

## I ran this command and it produced this output:
tox -e py3-cover

```
>       import ipdb; ipdb.set_trace()
E       ModuleNotFoundError: No module named 'ipdb'
```
in venv
despite the fact that it is installed. I think something with venv paths in 

## Certbot's behavior differed from what I expected because:
The Development Guide says that:
After running this command, certbot and development tools like ipdb, ipython, pytest, and tox are available in the shell where you ran the command. 
Actually, it's true for three commands except ipdb. It isn't present in terminal and interpreter can't see it.

## Here is a Certbot log showing the issue (if available):
###### Logs are stored in `/var/log/letsencrypt` by default. Feel free to redact domains, e-mail and IP addresses as you see fit.

## Here is the relevant nginx server block or Apache virtualhost for the domain I am configuring:

Well, the binary `ipdb3` is indeed present, although for some reason ipdb module itself isn't in the venv path so I still getting error:
```
>       import ipdb; ipdb.set_trace()
E       ModuleNotFoundError: No module named 'ipdb'
```

--- a/certbot/docs/contributing.rst
+++ b/certbot/docs/contributing.rst
@@ -75,7 +75,7 @@ latter by running:
 
    source venv/bin/activate
 
-After running this command, ``certbot`` and development tools like ``ipdb``,
+After running this command, ``certbot`` and development tools like ``ipdb3``,
 ``ipython``, ``pytest``, and ``tox`` are available in the shell where you ran
 the command. These tools are installed in the virtual environment and are kept
 separate from your global Python installation. This works by setting

